api/serializers.py

```diff
@@ -100,6 +100,7 @@ class PuzzleSerializer(serializers.ModelSerializer):
     chat_room = ChatRoomSerializer(required=False)
     tags = PuzzleTagSerializer(required=False, many=True)
     guesses = serializers.SerializerMethodField()
+    sheet = serializers.SerializerMethodField()
```

Comment:
nit: Should we call this "has_sheet" instead to clearly imply a boolean? I initially interpreted this as a sheet URL before seeing the test.

Reply:
Yeah we should, good call